include/z64.h

```diff
@@ -1065,7 +1065,7 @@ typedef struct GameState {
     /* 0x84 */ GameAlloc alloc;
     /* 0x98 */ u32 running;
     /* 0x9C */ u32 frames;
-    /* 0xA0 */ u32 unk_A0;
+    /* 0xA0 */ u32 isPreNMIStateRunning;
```

Comment:
I wonder if we couldn't use a simpler name like `inPreNMI` or `preNMIRunning`/`preNMIActive`

Reply:
Dropping "state" isn't possible imo, otherwise it'd be referencing PreNMI (the period between the PreNMI hardware interrupt and actual reset/NMI) instead of the state named PreNMI

`inPreNMIState` or just `isPreNMIState` may work?

Reply:
I was relying on the fact that this is a gameState field to omit the "state" part (eg. `gameState->inPreNMI`), but if you feel that's not clear enough then yeah we could go with `inPreNMIState`